File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/cake/CakeUpdateParam.java
package com.cakk.domain.mysql.dto.param.cake;

import java.util.List;

import com.cakk.domain.mysql.entity.cake.CakeCategory;
import com.cakk.domain.mysql.entity.user.User;

public record CakeUpdateParam(
	User owner,
	Long cakeId,
	String cakeImageUrl,
	List<CakeCategory> cakeCategories,
	List<String> tagNames
) {
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/cake/CakeUpdateParam.kt
package com.cakk.domain.mysql.dto.param.cake

import com.cakk.domain.mysql.entity.cake.CakeCategory
import com.cakk.domain.mysql.entity.user.User

data class CakeUpdateParam(
	val owner: User,
	val cakeId: Long,
	val cakeImageUrl: String,
	val cakeCategories: List<CakeCategory>,
	val tagNames: List<String>
)
